runtime/compiler/runtime/J9Profiler.hpp

```diff
@@ -633,20 +633,22 @@ class TR_BlockFrequencyInfo
    int32_t getFrequencyInfo(TR::Block *block, TR::Compilation *comp);
    int32_t getFrequencyInfo(TR_ByteCodeInfo &bci, TR::Compilation *comp, bool normalizeForCallers = true, bool trace = true);
    void    setCounterDerivationInfo(TR_BitVector **counterDerivationInfo) { _counterDerivationInfo = counterDerivationInfo; }
+   TR_BitVector ** getCounterDerivativeInfo() { return _counterDerivationInfo; }
```

Comment:
This isn't called anywhere - why is it exposed?

Reply:
That got out of my attention. No need to have that info. Cleaned up : Changes as per PR#2348 and Fix warnings